test/e2e/framework/pod/create.go

```diff
@@ -219,6 +244,18 @@ func MakeSecPod(ns string, pvclaims []*v1.PersistentVolumeClaim, inlineVolumeSou
 	podSpec.Spec.Containers[0].VolumeMounts = volumeMounts
 	podSpec.Spec.Containers[0].VolumeDevices = volumeDevices
 	podSpec.Spec.Volumes = volumes
-	podSpec.Spec.SecurityContext.SELinuxOptions = seLinuxLabel
+	podSpec.Spec.SecurityContext.SELinuxOptions = podInfo.seLinuxLabel
+
+	podSpec.Spec.NodeSelector = podInfo.nodeSelection.Selector
```

Comment:
Can we use SetNodeSelection() directly here instead?

Reply:
Done.